### PR TITLE
feat(domain): replace wuc-ovh-tabs with oui-header-tabs

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Before: Domains used `wuc-ovh-tabs`
Now: Domains uses oui-header-tabs and UI Router states 

MANAGER-2343

### CODE REVIEW

File: client/app/domain/dns/domain-dns.state.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const state = {
+  url: '/dns',
+  views: {
+    domainView: {
+      templateUrl: 'domain/dns/DNS.html',
```
Comment (frenautvh) [inline client/app/domain/dns/domain-dns.state.js]:
```javascript
import mytemplate from "foo/bar.html";
// ...
domainView: {
  template: mytemplate,
}
```
that way webpack will report errors if template is missing

Comment (cbourgois) [reply client/app/domain/dns/domain-dns.state.js]:
```javascript
import template from "foo/bar.html";
// ...
domainView: {
  template,
}
```

File: client/app/domain/domain.controller.js [hunk 2]
```diff
@@ -192,6 +194,10 @@ angular.module('App').controller(
         .finally(() => {
           this.loading.domainInfos = false;
           this.$scope.$broadcast('domain.refreshData.done');
+
+          if (!this.domain.isExpired) {
+            this.$state.go(`${this.getState()}.information`, this.$stateParams);
```
Comment (JeremyDec) [inline client/app/domain/domain.controller.js]: `return` ?

Comment (marie-j) [reply client/app/domain/domain.controller.js]: As we're in the `finally` hook, the value will never be used (when chaining we'll get the result from the `then`/`catch` hook )

**Pull request merged**